docs/features/techdocs/configuring-ci-cd.md

```diff
@@ -96,3 +102,66 @@ npx @techdocs/cli publish --publisher-type <awsS3|googleGcs> --storage-name <buc
 
 The updated TechDocs site built in this workflow is now ready to be served by
 the TechDocs plugin in your Backstage app.
+
+## Example: GitHub Actions CI and AWS S3
+
+Here is an example workflow using GitHub Actions CI and AWS S3 storage. You can
+use any CI and other cloud storage providers - Google Cloud Storage and Azure
+Blob Storage.
```

Comment:
To avoid tying information on a page about CI/CD too closely to supported storage providers, maybe this should end with something like "and any other TechDocs supported cloud storage provider" with the "TechDocs supported cloud storage provider" linked here:

https://backstage.io/docs/features/techdocs/techdocs-overview#platforms-supported 

?

Reply:
Perfect, yes!